ush/generate_FV3LAM_wflow.sh

```diff
@@ -767,6 +785,27 @@ if [ "${DO_BUFRSND}" = "TRUE" ]; then
   fi
 fi
 
+
+if [ "${DO_MINMAXT}" = "TRUE" ]; then
+#
+#
+# Resolve the target directory that the FIXbufrsnd symlink points to
+#
+  ln_vrfy -fsn "$MINMAX_BINS" "$FIX_MINMAXT"
+
+  path_resolved=$( readlink -m "$FIX_MINMAXT" )
+  if [ ! -d "${path_resolved}" ]; then
+    print_err_msg_exit "\
+  Missing link to FIXsmokedust
+  RUN_ENVIR = \"${RUN_ENVIR}\"
+  FIX_MINMAXT = \"$FIX_MINMAXT\"
+  path_resolved = \"${path_resolved}\"
+  Please ensure that path_resolved is an existing directory and then rerun
+  the experiment generation script."
+  fi
+fi
```

Comment:
This looks like a temporary patch to me. I think we can discuss on this further.